compose/development/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM  ubuntu:xenial
+LABEL maintainer achillesrasquinha@gmail.com
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update --fix-missing
+RUN apt-get install -y --no-install-recommends \
+    	apt-transport-https \
+    	gcc \
+    	git \
+    	python3-dev \ 
+    	python3-pip \
+    	python3-tk \
+    	software-properties-common \ 
+    	graphviz-dev \
+    && echo debconf shared/accepted-oracle-license-v1-1 select true | debconf-set-selections 
+
+RUN add-apt-repository -y ppa:webupd8team/java \ 
+    && apt-get update \
+    && apt-get install -y --no-install-recommends oracle-java8-installer \
+    && apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E084DAB9 \
+    && add-apt-repository 'deb [arch=amd64,i386] https://cran.rstudio.com/bin/linux/ubuntu xenial/' \
+    && apt-get update \
+    && apt-get install -y r-base 
+
+COPY ./requirements /requirements
+
+RUN pip3 install --upgrade pip
+RUN pip3 install setuptools wheel \
+    && pip3 install numpy pyyaml \
+    && pip3 install --no-binary javabridge -r /requirements/development.txt  -r /requirements/production.txt
```

Comment:
add all dependencies in development.txt, production.txt will be the subset of it.

Reply:
@rupav  that has one issue. Any new dependency then has to be added in both the files.
In this only one file u need to add.

Reply:
Yes, But development.txt implies that all dependencies in it should be sufficient to start on local setup for new developers. Not everyone will directly jump into docker for setup.

Reply:
True. Will do it